config.yaml

```diff
@@ -0,0 +1,4 @@
+map: ../maps/matfyz.osm
```

Comment:
@Robotics-DAI-FMFI-UK/robotour-smely-zajko-team Can you think of another configuration options that should make it to this file? I believe it would be beneficial not to recompile anything unless some functionality has actually changed.